packages/vb-decompiler-lite.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,17 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+try{
+
+    $toolName = 'VB Decompiler Lite'
```

Comment:
I think we should use the executable name as tool name as documented in https://github.com/mandiant/VM-Packages/wiki/Coding-Conventions#toolname
```suggestion
    $toolName = 'VB Decompiler'
```